cmd/juju/application/store/charmadapter_test.go

```diff
@@ -299,7 +155,7 @@ func (s *resolveSuite) setupMocks(c *gc.C) *gomock.Controller {
 
 func (s *resolveSuite) expectCharmResolutionCall(curl *charm.URL, out csparams.Channel, err error) {
```

Comment:
Replace "out csparams.Channel" as an arg. It's a charmstore specific old style channel.